### DISCUSSION TITLE
Make new language available on website?

### DISCUSSION THREAD

**Discussion by themortenbay** [DivinumOfficium/divinum-officium#3019] [Q&A] [answered]
Hiya.
So my Danish translation has now been mergen into the Divinum Officium project proper. Does something have to be changed/ applied for the translation to appear as an option on the website - or is the website simply lacking a little behind the github repository?
Thanks in advance. :)
In Xto

**Comment by mbab** [comment, marked as answer]
Time is also account. But You need to add new language in file `web/www/horas/horas.dialog` line 33.

**Comment by Vitalij-82** [comment]
Добрый день.возможно ли в дивинум оффициум добавить русский язык,буду весьма признателен вам.

**Reply by rl1234567890** [reply]
К сожалению, я не говорю по-русски. Я использовал Google, чтобы перевести это. То, что вы просите, технически возможно, но эту работу должен выполнить кто-то, обладающий достаточным знанием русского языка, латыни и литургии.

**Reply by Augustinus-Altovadensis** [reply]
Тут нет вопроса, возможно ли это или нет.  Возможно, DivinumOfficium.com поддерживает UFT8, я только что попробовал. Вопрос лишь в том, _кто_ это сделает. Это огромный объем работы, для каждого ритуала это немного по-разному... 

Так что спросите себя: настолько ли мне нужен этот перевод, **чтобы я его сделал?** Тогда мы будем рады помочь вам с вопросом "как".

**Reply by Vitalij-82** [reply]
Спасибо большое за ответ.Да,перевод на русский язык очень нужен,так как я не владею латынью и с изучением языка у меня большая проблема и я очень люблю прежнюю форму бревиария а не пособорную 2Ватиканского Собора.Буду вам очень благодарен,да поможет вам Бог в ваших трудах.

**Comment by Vitalij-82** [comment]
Спасибо большое за ответ.Да,перевод на русский язык очень нужен,так как я
не владею латинским и с изучением языка у меня большая проблема и я очень
люблю прежнюю форму бревиария а не пособорную 2Ватиканского Собора.Буду вам
очень благодарен,да поможет вам Бог в ваших трудах.

пт, 10 нояб. 2023 г., 15:09 Jiří Emmer ***@***.***>:
